src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -847,7 +848,8 @@ return function ($context) {
         case 'json':
             return $context->res->json(["type" => "This is a JSON response"]);
         case 'binary':
-            return $context->res->binary(InputFile::withPath('file.png'));
+            $fileContent = file_get_contents('file.png');
```

Comment:
Let's check with Steven for confirmation. I think it would be better to have a snippet that can do relative path. Because with absolute you would need to `/usr/local/server/src/function/[MY_PATH]`, which is very hard to remember

Reply:
> Let's check with Steven for confirmation. I think it would be better to have a snippet that can do relative path. Because with absolute you would need to `/usr/local/server/src/function/[MY_PATH]`, which is very hard to remember

Yep. I think that'd fall on other runtimes too, right?

Reply:
This gets very tricky due to how some runtimes will include the assets in the source vs others won't.

The code sample we currently have is essentially relative. We should just stick to that and ensure we use a consistent path for all examples.